app/views/pages/dataset.html.haml

```diff
@@ -10,7 +10,7 @@
           = image_tag 'reports/print.svg'
           %span= t('report.print')
 
-    .page#establishment_shot{ data: { area: @area.area, host: APP_CONFIG[:api_url], max_year: Setting::MAX_YEAR, min_year: Setting::MIN_YEAR, default_year: Setting::DEFAULT_YEAR, scenario_id: Current.setting.api_session_id, time: @time, non_energy: params[:non_energy] } }
+    .page#establishment_shot{ data: { area: @area.area, host: APP_CONFIG[:api_url], max_year: Setting::MAX_YEAR, min_year: Setting::MIN_YEAR, default_year: Setting::DEFAULT_YEAR, scenario_id: @scenario_id, time: @time, non_energy: params[:non_energy] } }
```

Comment:
Not necessary for this pull request: If you want to quiet this linting error, my advice would be to move the factsheet options into a helper method in `PagesHelper`. It would make the template a little clearer, since the long list of options aren't needed to understand the template itself.

You'd have something like this:

```ruby
module PagesHelper
  # ...

  def co2_factsheet_options
    {
      area: @area.area,
      default_year: Setting::DEFAULT_YEAR,
      host: APP_CONFIG[:api_url],
      max_year: Setting::MAX_YEAR,
      min_year: Setting::MIN_YEAR,
      scenario_id: @scenario_id,
      time: @time
    }
  end
end
```

... and from the HAML template it would be used as:

```haml
.page#establishment_shot{ data: co2_factsheet_options }
  -# etc
```